zer0-os/zOS
# refactor(favorites): use channel label management to handle favorite room tagging

### What does this do?
- replaces specific favorites functions/calls with label management functions/calls to handle favorite room tagging

### Why are we making this change?
- now that we have label management in place we can consolidate the handling of room tagging.

### How do I test this?
- run tests as usual.
- run the ui and test favorite room tagging is working as expected.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

https://github.com/user-attachments/assets/5f3321b0-fbc4-40c6-9508-0488d49415b9
